lib/llvmopencl/Workgroup.cc

```diff
@@ -1362,7 +1392,12 @@ Workgroup::createGridLauncher(Function *KernFunc, Function *WGFunc,
   LLVMBuildRetVoid(Builder);
 
   InlineFunctionInfo IFI;
-  InlineFunction(dyn_cast<CallInst>(llvm::unwrap(Call)), IFI);
+  InlineFunction(
+#ifndef LLVM_OLDER_THAN_11_0
+      *
```

Comment:
Also here it looks strange :) IMHO just replicating the statement is fine and more readable.